package/main/daml/Daml.Finance.Interface.Types.Date/daml.yaml

```
@@ -4,9 +4,10 @@
 sdk-version: 2.5.0
 name: daml-finance-interface-types-date
 source: daml
-version: 1.0.0
+version: 1.0.1
```

I am afraid we'll need

```suggestion
version: 2.0.0
```

as we are introducing breaking changes

---

Is there a way to change that automatically, given that I already ran `packell` via `make update-packages` ?

---

Yes - in the file `package/packages.yaml`, make the following updates to the package config of `Daml.Finance.Interface.Types.Date` :

current config: `increment-version: patch`
new config: `increment-version: major`

Then re-run `make update-packages`.